qiita_db/study.py

```diff
@@ -617,11 +635,14 @@ def investigation(self):
         -------
         Investigation id
         """
-        conn_handler = SQLConnectionHandler()
-        sql = ("SELECT investigation_id FROM qiita.investigation_study WHERE "
-               "study_id = %s")
-        inv = conn_handler.execute_fetchone(sql, (self._id, ))
-        return inv[0] if inv is not None else inv
+        with TRN:
+            sql = """SELECT investigation_id FROM qiita.investigation_study
```

Comment:
I don't think it is needed. If it exists, it will be only one. The `inv[0][0]` is because if it doesn't exist, `TRN.execute_fetchindex` returns an empty list, but if there is a value it will return `[[inv_id]]`, and doesn't matter how I change the query, since this is because the transaction internally is doing a fetchall.